src/UIAutomation/Microsoft.UI.UIAutomation/Standins.cpp

```diff
@@ -523,6 +523,14 @@ namespace winrt::Microsoft::UI::UIAutomation::implementation
         return result;
     }
 
+    winrt::AutomationRemoteElement AutomationRemoteElement::BuildUpdatedCache(const winrt::AutomationRemoteCacheRequest& cacheRequest)
```

Comment:
I am not sure if this is the best name. It follows the API name (which is fine/proper for the abstraction) but it does not sound as appropriate here...

How about `GetUpdatedElement`? Or something else?

Reply:
Hmm, what about `GetUpdatedCacheElement` ?

While I suppose keeping this as `BuildUpdatedCache` wouldn't be a big deal (as you said, that's the current classic COM API name for effectively the same thing) and most people using this library will likely have already used the classic COM API, it might be an opportunity to improve.

Reply:
I am good with that name change. 👍

I will make it on both levels (abstraction and standins). If you are not good with one of the places, happy to hear and make other changes if needed (conclusion: naming/describing things is difficult). :)